index.html

```diff
@@ -10,28 +10,239 @@
     <title>TripHouse Project</title>
 </head>
 <body>
-<section class="offer-container">
-    <h3 class="offer-title">What do we offer</h3>
-    <div class="offer-title-line"></div>
-    <ul class="offer-blocks-wrapper">
-        <li class="block">
-            <img class="img-container" src="./img/project_images/support.png" alt="image:support">
-            <p class="block-info">Support 24/7</p>
-        </li>
-        <li class="block">
-            <img class="img-container" src="./img/project_images/communicate.png" alt="image:communicate">
-            <p class="block-info">Communicate directly</p>
-        </li>
-        <li class="block">
-            <img class="img-container" src="./img/project_images/book.png" alt="image:book-online">
-            <p class="block-info">Book online</p>
-        </li>
-        <li class="block">
-            <img class="img-container" src="./img/project_images/review.png" alt="image:guests-reviews">
-            <p class="block-info">Real guest reviews</p>
-        </li>
-    </ul>
-</section>
+<main>
+    <section class="main">
+        <div class="main-container">
+            <div class="main-header">
```

Comment:
You should use `<header>` here :)
![image](https://user-images.githubusercontent.com/25488411/157955902-e94d8c4b-e170-4ab2-aee4-99224f9b7c48.png)

Reply:
Done.